configs/registration_local.properties

```diff
@@ -5,7 +5,11 @@ registration.port=8085
 registration.registrationCredential.accountId=registration_service@notary
 registration.registrationCredential.pubkeyPath=deploy/iroha/keys/registration_service@notary.pub
 registration.registrationCredential.privkeyPath=deploy/iroha/keys/registration_service@notary.priv
-# --------- Iroha --------------
+registration.clientStorageAccount=client_accounts@notary
+registration.brvsAccount=brvs@brvs
+registration.primaryPubkeyPath=deploy/iroha/keys/registration_service_fake@notary.pub
```

Comment:
what is fake?

Reply:
the key that is used for the first step of a client registration, the user account created with this 'fake' key first

Reply:
I think it is better to rename to initial or introductory. I don't like word `fake`. 😄

Reply:
Okay, I renamed it to 'primary'